src/main/resources/reference-data/dwpAddresses.json

```diff
@@ -117,19 +117,6 @@
         "postCode": "WV99 1AG"
       }
     },
-    {
-      "code": "10",
```

Comment:
Will removing this create problems for any case that is already set to PIP 10?

Reply:
Thanks, good to check, let me check with Hemingway, I think we will not be having PIP 10 cases